modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/data_set/filters/modals/selection_filter/source_type/ApiRestApplication.tsx

```diff
@@ -100,11 +107,65 @@ function ApiRestApplication({
 		return true;
 	};
 
+	async function getAPIValues(source: string) {
```

Comment:
Can we put this inline in `getSourceItems`, to avoid naming?

Reply:
sure, done!